### ISSUE TITLE
Notepad++ freezes when file is synced by pCloud

### ISSUE THREAD

**Issue by yanko397** [p0358/notepadpp-CodeStats#18]
If the Code::Stats plugin is installed, notepad++ will freeze after you edit and save a file in a directory that is synchronized by the pCloud desktop app. The freeze happens in the exact moment when pCloud starts syncing. This seems to only happen with pCloud, no other cloud provider i've tested.

The freeze itself is not a real freeze, as all buttons in notepad++ flicker rapidly when you hover over them, but you cannot interact with notepad++ anymore and have to close it via the task manager.

**Comment by p0358** [comment]
@yanko397 Thanks for reporting, this case should be very useful for testing.
And I assume the freeze doesn't happen with pCloud syncing, but without the plugin?
And with both installed, does it always freeze, or just sometimes?

**Comment by yanko397** [comment]
Yes, without the plugin everything works perfectly. With the plugin installed it freezes every time.
